lib/__tests__/expected.yml

```diff
@@ -568,7 +234,7 @@ Resources:
                 - Name: Artifact_Build_Build
               Name: TestHelloLinux
               OutputArtifacts:
-                - Name: Artifact_delivlibtestCodeCommitPipelineHelloLinux01786D9D
+                - Name: Artifact_c883e6647f907b1eb255846397acb348c18b48b3a2
```

Comment:
The artifact name change is what we get when we go from `node.uniqueId` => `node.addr`